### PR TITLE
Remove icon and add command palette to sync data

### PR DESCRIPTION
**Pull request by sywhb**
* Remove the icon and add a command palette to sync data
* Remove background sync
* Add a link to template references in settings
* Update default template and add YAML front matter metadata

### CODE REVIEW

**Comment by jacksonh** [comment]
Oh command thing is cool.

File: src/main.ts [hunk 1]
```diff
@@ -35,39 +34,43 @@ interface Settings {
...
-  labels: {{#labels}}[[{{{name}}}]]{{/labels}}
-  {{/labels.length}}
-  date_saved: {{{dateSaved}}}`,
+  articleTemplate: `---
+{{#author}}
+author: {{{author}}}
+{{/author}}
+{{#labels.length}}
+tags:
+{{#labels}}  - {{{name}}}
```
Comment (jacksonh) [inline src/main.ts:57]: Oh this is nice, so its a yaml list. One issue I wonder about, is I think tags have to be a single word in Obsidian. Will see what happens with some of my two word labels.

**Comment by jacksonh** [comment]
I think the only thing we are missing is a tag like #Omnivore that will link all saved pages together.

**Review by jacksonh — APPROVED** [review]

File: src/main.ts (hunk 1 above)
Comment (sywhb) [reply src/main.ts:57]: Good catch.

According to Obsidian docs:
```
Spaces are not allowed in tags. So, to differentiate two or more words in a tag, you can use these case styles/formats:

camelCase: [#twoWords](https://publish.obsidian.md/#twoWords)
PascalCase: [#TwoWords](https://publish.obsidian.md/#TwoWords)
snake_case: [#two_words](https://publish.obsidian.md/#two_words)
kebab-case: [#two-words](https://publish.obsidian.md/#two-words)
```

So I will use camel case

**Pull request merged**